### ISSUE TITLE
'Could not initialize class' Error

### ISSUE THREAD

**Issue by momo9** [postalservice14/vuejs-plugin#2]
Hello, I encountered an error after installing this plugin.

My WebStorm version is 10.0.3, vuejs-plugin version is 1.0.1.

I've tried to change the JDK bundled with WebStorm (from 1.6 to 1.8), however it made no sense.

![screen shot 2015-12-24 at 9 58 22 am](https://cloud.githubusercontent.com/assets/6291202/11988162/12bb91fc-aa26-11e5-984e-bce76abc5d5a.png)

**Comment by postalservice14** [comment]
Interesting.  I'm still new to plugin development.  I'm running WebStorm 11.0.3 and the plugin appears to be working for me. :-/

Maybe check your IDE log files to see if there is a stacktrace or something?  Thanks for helping me track this down. https://intellij-support.jetbrains.com/hc/en-us/articles/206827517-Locating-IDE-log-files

**Comment by momo9** [comment]
I hope that the log can help you.

```
2016-01-03 23:46:15,061 [1434642]   INFO -   #com.intellij.AbstractBundle - Cannot load resource bundle from *.properties file, falling back to slow class loading: messages.WebIdeBundle 
2016-01-03 23:46:15,062 [1434643]   INFO - j.ide.plugins.PluginDescriptor - Cannot find plugin com.intellij resource-bundle: messages.WebIdeBundle 
2016-01-03 23:46:32,708 [1452289]   INFO - Types.impl.FileTypeManagerImpl - FileTypeManager: 1 auto-detected files
Elapsed time on auto-detect: 61 ms 
2016-01-03 23:46:32,722 [1452303]   INFO - org.jetbrains.io.BuiltInServer - web server stopped 
2016-01-03 23:46:32,903 [1452484]   INFO - il.indexing.FileBasedIndexImpl - START INDEX SHUTDOWN 
2016-01-03 23:46:32,934 [1452515]   INFO - il.indexing.FileBasedIndexImpl - END INDEX SHUTDOWN 
2016-01-03 23:46:32,934 [1452515]   INFO - stubs.SerializationManagerImpl - START StubSerializationManager SHUTDOWN 
2016-01-03 23:46:32,934 [1452515]   INFO - stubs.SerializationManagerImpl - END StubSerializationManager SHUTDOWN 
2016-01-03 23:46:33,060 [1452641]   WARN - api.vfs.impl.local.FileWatcher - Watcher terminated with exit code 0 
2016-01-03 23:46:33,060 [1452641]   INFO - newvfs.persistent.PersistentFS - VFS dispose started 
2016-01-03 23:46:33,065 [1452646]   INFO - newvfs.persistent.PersistentFS - VFS dispose completed 
2016-01-03 23:46:33,101 [1452682]   INFO -        #com.intellij.idea.Main - ------------------------------------------------------ IDE SHUTDOWN ------------------------------------------------------ 
2016-01-03 23:46:35,471 [      0]   INFO -        #com.intellij.idea.Main - ------------------------------------------------------ IDE STARTED ------------------------------------------------------ 
2016-01-03 23:46:35,517 [     46]   INFO -        #com.intellij.idea.Main - IDE: WebStorm (build #WS-141.1237, 22 May 2015 00:00) 
2016-01-03 23:46:35,518 [     47]   INFO -        #com.intellij.idea.Main - OS: Mac OS X (10.10.5, x86_64) 
2016-01-03 23:46:35,518 [     47]   INFO -        #com.intellij.idea.Main - JRE: 1.6.0_65-b14-466.1-11M4716 (Apple Inc.) 
2016-01-03 23:46:35,518 [     47]   INFO -        #com.intellij.idea.Main - JVM: 20.65-b04-466.1 (Java HotSpot(TM) 64-Bit Server VM) 
2016-01-03 23:46:35,525 [     54]   INFO -        #com.intellij.idea.Main - JVM Args: -Dfile.encoding=UTF-8 -XX:+UseConcMarkSweepGC -XX:SoftRefLRUPolicyMSPerMB=50 -ea -Dsun.io.useCanonCaches=false -Djava.net.preferIPv4Stack=true -Xverify:none -Xbootclasspath/a:../lib/boot.jar -Xms128m -Xmx750m -XX:MaxPermSize=350m -XX:ReservedCodeCacheSize=225m -XX:+UseCompressedOops -Djb.vmOptionsFile=/Applications/WebStorm.app/Contents/bin/webstorm.vmoptions -Didea.java.redist=NoJavaDistribution -Didea.home.path=/Applications/WebStorm.app/Contents -Didea.executable=webstorm -Didea.platform.prefix=WebStorm -Didea.paths.selector=WebStorm10 
2016-01-03 23:46:35,583 [    112]   INFO -        #com.intellij.idea.Main - JNA library loaded (64-bit) in 58 ms 
2016-01-03 23:46:35,593 [    122]   INFO -        #com.intellij.idea.Main - initializing environment 
2016-01-03 23:46:35,596 [    125]   INFO - .intellij.util.EnvironmentUtil - loading shell env: /usr/local/bin/zsh -l -i -c '/Applications/WebStorm.app/Contents/bin/printenv.py' '/private/var/folders/f7/k95pwqj53qb4jbgy6ddbkby40000gn/T/intellij-shell-env0.tmp' 
2016-01-03 23:46:36,393 [    922]   INFO - .intellij.util.EnvironmentUtil - shell environment loaded (34 vars) 
2016-01-03 23:46:37,598 [   2127]   INFO - llij.ide.plugins.PluginManager - Cannot find optional descriptor js-nashorn-support.xml 
2016-01-03 23:46:38,706 [   3235]   INFO - llij.ide.plugins.PluginManager - 52 plugins initialized in 1924 ms 
2016-01-03 23:46:38,708 [   3237]   INFO - llij.ide.plugins.PluginManager - Loaded bundled plugins: ASP (0.1), AngularJS (141.1237), CSS Support, CVS Integration (11), CoffeeScript (2.0), Cucumber.js (141.1237), Dart (141.1237), EJS (141.1237), EditorConfig (1.0), File Watchers (141.1237), Gherkin (999.999), Git Integration (8.1), GitHub, HAML, HTML Tools (2.0), Handlebars/Mustache (141.1237), IDEA CORE, IntelliLang (8.0), JSTestDriver Plugin (141.1237), Jade (141.1237), Java Server Pages Integration (1.0), JavaScript Debugger (1.0), JavaScript Intention Power Pack (0.9.4), JavaScript Support (1.0), Karma (141.1237), LESS support, LiveEdit (999.0), Meteor (141.1237), NodeJS (141.1237), Perforce Integration (2.0), PhoneGap/Cordova Plugin (141.1237), Polymer & Web Components (141.1237), QuirksMode, REST Client, Refactor-X (2.01), Remote Hosts Access (0.1), SASS support, Spy-js (141.1237), Stylus support (141.1237), Subversion Integration (1.1), Task Management (1.0), Terminal (0.1), TextMate bundles support (VERSION), W3C Validators (2.0), XPathView + XSLT Support (4), XSLT-Debugger (1.4), YAML, hg4idea (10.0) 
2016-01-03 23:46:38,708 [   3237]   INFO - llij.ide.plugins.PluginManager - Loaded custom plugins: Bootstrap 3 (2.1), Markdown (0.9.7), SvgViewer 2 (2.0.3), Vue.js (1.0.1) 
2016-01-03 23:46:39,358 [   3887]   INFO - ellij.util.io.PagedFileStorage - lower=100; upper=200; buffer=10; max=756744192 
2016-01-03 23:46:39,437 [   3966]   INFO - api.vfs.impl.local.FileWatcher - Starting file watcher: /Applications/WebStorm.app/Contents/bin/fsnotifier 
2016-01-03 23:46:39,467 [   3996]   INFO - api.vfs.impl.local.FileWatcher - Native file watcher is operational. 
2016-01-03 23:46:39,690 [   4219]  ERROR - nsions.impl.ExtensionPointImpl - org/vuejs/VueJSFileFactory : Unsupported major.minor version 52.0 [Plugin: Vue.js] 
com.intellij.openapi.extensions.impl.PicoPluginExtensionInitializationException: org/vuejs/VueJSFileFactory : Unsupported major.minor version 52.0 [Plugin: Vue.js]
	at com.intellij.openapi.extensions.impl.ExtensionComponentAdapter.getComponentInstance(ExtensionComponentAdapter.java:96)
	at com.intellij.openapi.extensions.impl.ExtensionComponentAdapter.getExtension(ExtensionComponentAdapter.java:119)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.processAdapters(ExtensionPointImpl.java:249)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.getExtensions(ExtensionPointImpl.java:194)
	at com.intellij.openapi.extensions.Extensions.getExtensions(Extensions.java:101)
	at com.intellij.openapi.extensions.Extensions.getExtensions(Extensions.java:87)
	at com.intellij.openapi.extensions.ExtensionPointName.getExtensions(ExtensionPointName.java:49)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.initStandardFileTypes(FileTypeManagerImpl.java:268)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.<init>(FileTypeManagerImpl.java:230)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:39)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:27)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:513)
	at org.picocontainer.defaults.InstantiatingComponentAdapter.newInstance(InstantiatingComponentAdapter.java:193)
	at org.picocontainer.defaults.ConstructorInjectionComponentAdapter$1.run(ConstructorInjectionComponentAdapter.java:220)
	at org.picocontainer.defaults.ThreadLocalCyclicDependencyGuard.observe(ThreadLocalCyclicDependencyGuard.java:53)
	at org.picocontainer.defaults.ConstructorInjectionComponentAdapter.getComponentInstance(ConstructorInjectionComponentAdapter.java:248)
	at com.intellij.util.pico.ConstructorInjectionComponentAdapter.getComponentInstance(ConstructorInjectionComponentAdapter.java:58)
	at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentConfigComponentAdapter$1.getComponentInstance(ComponentManagerImpl.java:550)
	at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentConfigComponentAdapter.getComponentInstance(ComponentManagerImpl.java:610)
	at com.intellij.util.pico.DefaultPicoContainer.getLocalInstance(DefaultPicoContainer.java:245)
	at com.intellij.util.pico.DefaultPicoContainer.getComponentInstance(DefaultPicoContainer.java:211)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.createComponent(ComponentManagerImpl.java:125)
	at com.intellij.openapi.application.impl.ApplicationImpl.createComponent(ApplicationImpl.java:354)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.a(ComponentManagerImpl.java:116)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.init(ComponentManagerImpl.java:87)
	at com.intellij.openapi.components.impl.stores.ApplicationStoreImpl.load(ApplicationStoreImpl.java:101)
	at com.intellij.openapi.application.impl.ApplicationImpl.load(ApplicationImpl.java:499)
	at com.intellij.openapi.application.impl.ApplicationImpl.load(ApplicationImpl.java:481)
	at com.intellij.idea.IdeaApplication.run(IdeaApplication.java:170)
	at com.intellij.idea.MainImpl$1$1$1.run(MainImpl.java:64)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:209)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:715)
	at java.awt.EventQueue.access$400(EventQueue.java:82)
	at java.awt.EventQueue$2.run(EventQueue.java:676)
	at java.awt.EventQueue$2.run(EventQueue.java:674)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:86)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:685)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:362)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:296)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:211)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:196)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:188)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:122)
Caused by: com.intellij.diagnostic.PluginException: org/vuejs/VueJSFileFactory : Unsupported major.minor version 52.0 [Plugin: Vue.js]
	at com.intellij.ide.plugins.cl.PluginClassLoader.b(PluginClassLoader.java:130)
	at com.intellij.ide.plugins.cl.PluginClassLoader.a(PluginClassLoader.java:77)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:66)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:247)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:249)
	at com.intellij.openapi.extensions.impl.ExtensionComponentAdapter.loadImplementationClass(ExtensionComponentAdapter.java:156)
	at com.intellij.openapi.extensions.impl.ExtensionComponentAdapter.getComponentImplementation(ExtensionComponentAdapter.java:66)
	at com.intellij.openapi.extensions.impl.ExtensionComponentAdapter.getComponentInstance(ExtensionComponentAdapter.java:73)
	... 45 more
Caused by: java.lang.UnsupportedClassVersionError: org/vuejs/VueJSFileFactory : Unsupported major.minor version 52.0
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClassCond(ClassLoader.java:637)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:621)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:471)
	at com.intellij.util.lang.UrlClassLoader._defineClass(UrlClassLoader.java:259)
	at com.intellij.util.lang.UrlClassLoader.defineClass(UrlClassLoader.java:255)
	at com.intellij.util.lang.UrlClassLoader._findClass(UrlClassLoader.java:231)
	at com.intellij.ide.plugins.cl.PluginClassLoader.b(PluginClassLoader.java:124)
	... 53 more
Caused by:

com.intellij.diagnostic.PluginException: org/vuejs/VueJSFileFactory : Unsupported major.minor version 52.0 [Plugin: Vue.js]
	at com.intellij.ide.plugins.cl.PluginClassLoader.b(PluginClassLoader.java:130)
	at com.intellij.ide.plugins.cl.PluginClassLoader.a(PluginClassLoader.java:77)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:66)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:247)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:249)
	at com.intellij.openapi.extensions.impl.ExtensionComponentAdapter.loadImplementationClass(ExtensionComponentAdapter.java:156)
	at com.intellij.openapi.extensions.impl.ExtensionComponentAdapter.getComponentImplementation(ExtensionComponentAdapter.java:66)
	at com.intellij.openapi.extensions.impl.ExtensionComponentAdapter.getComponentInstance(ExtensionComponentAdapter.java:73)
	at com.intellij.openapi.extensions.impl.ExtensionComponentAdapter.getExtension(ExtensionComponentAdapter.java:119)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.processAdapters(ExtensionPointImpl.java:249)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.getExtensions(ExtensionPointImpl.java:194)
	at com.intellij.openapi.extensions.Extensions.getExtensions(Extensions.java:101)
	at com.intellij.openapi.extensions.Extensions.getExtensions(Extensions.java:87)
	at com.intellij.openapi.extensions.ExtensionPointName.getExtensions(ExtensionPointName.java:49)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.initStandardFileTypes(FileTypeManagerImpl.java:268)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.<init>(FileTypeManagerImpl.java:230)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:39)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:27)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:513)
	at org.picocontainer.defaults.InstantiatingComponentAdapter.newInstance(InstantiatingComponentAdapter.java:193)
	at org.picocontainer.defaults.ConstructorInjectionComponentAdapter$1.run(ConstructorInjectionComponentAdapter.java:220)
	at org.picocontainer.defaults.ThreadLocalCyclicDependencyGuard.observe(ThreadLocalCyclicDependencyGuard.java:53)
	at org.picocontainer.defaults.ConstructorInjectionComponentAdapter.getComponentInstance(ConstructorInjectionComponentAdapter.java:248)
	at com.intellij.util.pico.ConstructorInjectionComponentAdapter.getComponentInstance(ConstructorInjectionComponentAdapter.java:58)
	at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentConfigComponentAdapter$1.getComponentInstance(ComponentManagerImpl.java:550)
	at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentConfigComponentAdapter.getComponentInstance(ComponentManagerImpl.java:610)
	at com.intellij.util.pico.DefaultPicoContainer.getLocalInstance(DefaultPicoContainer.java:245)
	at com.intellij.util.pico.DefaultPicoContainer.getComponentInstance(DefaultPicoContainer.java:211)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.createComponent(ComponentManagerImpl.java:125)
	at com.intellij.openapi.application.impl.ApplicationImpl.createComponent(ApplicationImpl.java:354)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.a(ComponentManagerImpl.java:116)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.init(ComponentManagerImpl.java:87)
	at com.intellij.openapi.components.impl.stores.ApplicationStoreImpl.load(ApplicationStoreImpl.java:101)
	at com.intellij.openapi.application.impl.ApplicationImpl.load(ApplicationImpl.java:499)
	at com.intellij.openapi.application.impl.ApplicationImpl.load(ApplicationImpl.java:481)
	at com.intellij.idea.IdeaApplication.run(IdeaApplication.java:170)
	at com.intellij.idea.MainImpl$1$1$1.run(MainImpl.java:64)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:209)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:715)
	at java.awt.EventQueue.access$400(EventQueue.java:82)
	at java.awt.EventQueue$2.run(EventQueue.java:676)
	at java.awt.EventQueue$2.run(EventQueue.java:674)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:86)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:685)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:362)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:296)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:211)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:196)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:188)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:122)
Caused by: java.lang.UnsupportedClassVersionError: org/vuejs/VueJSFileFactory : Unsupported major.minor version 52.0
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClassCond(ClassLoader.java:637)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:621)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:471)
	at com.intellij.util.lang.UrlClassLoader._defineClass(UrlClassLoader.java:259)
	at com.intellij.util.lang.UrlClassLoader.defineClass(UrlClassLoader.java:255)
	at com.intellij.util.lang.UrlClassLoader._findClass(UrlClassLoader.java:231)
	at com.intellij.ide.plugins.cl.PluginClassLoader.b(PluginClassLoader.java:124)
	... 53 more
2016-01-03 23:46:39,694 [   4223]  ERROR - nsions.impl.ExtensionPointImpl - WebStorm 10.0.3  Build #WS-141.1237 
2016-01-03 23:46:39,694 [   4223]  ERROR - nsions.impl.ExtensionPointImpl - JDK: 1.6.0_65 
2016-01-03 23:46:39,694 [   4223]  ERROR - nsions.impl.ExtensionPointImpl - VM: Java HotSpot(TM) 64-Bit Server VM 
2016-01-03 23:46:39,694 [   4223]  ERROR - nsions.impl.ExtensionPointImpl - Vendor: Apple Inc. 
2016-01-03 23:46:39,694 [   4223]  ERROR - nsions.impl.ExtensionPointImpl - OS: Mac OS X 
2016-01-03 23:46:40,277 [   4806]   INFO - lij.diagnostic.DebugLogManager - Set DEBUG for the following categories: [] 
2016-01-03 23:46:40,816 [   5345]   INFO - rains.ide.BuiltInServerManager - built-in server started, port 63342 
2016-01-03 23:46:41,990 [   6519]   INFO - pl.stores.ApplicationStoreImpl - 78 application components initialized in 3234 ms 
2016-01-03 23:46:42,012 [   6541]   INFO - .intellij.idea.IdeaApplication - App initialization took 7344 ms 
2016-01-03 23:46:42,283 [   6812]   WARN - om.intellij.util.ProfilingUtil - Profiling agent is not enabled. Add -agentlib:yjpagent to idea.vmoptions if necessary to profile IDEA. 
2016-01-03 23:46:43,205 [   7734]   INFO - roject.impl.ProjectManagerImpl - com.intellij.diagnostic.PluginException: kotlin/PropertyMetadata [Plugin: Vue.js] 
com.intellij.ide.plugins.PluginManager$StartupAbortedException: com.intellij.diagnostic.PluginException: kotlin/PropertyMetadata [Plugin: Vue.js]
	at com.intellij.ide.plugins.PluginManager.handleComponentError(PluginManager.java:245)
	at com.intellij.openapi.components.impl.PlatformComponentManagerImpl.handleInitComponentError(PlatformComponentManagerImpl.java:39)
	at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentsRegistry.a(ComponentManagerImpl.java:441)
	at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentsRegistry.a(ComponentManagerImpl.java:417)
	at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentsRegistry.access$000(ComponentManagerImpl.java:403)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.a(ComponentManagerImpl.java:108)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.init(ComponentManagerImpl.java:87)
	at com.intellij.openapi.project.impl.ProjectImpl.init(ProjectImpl.java:284)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.a(ProjectManagerImpl.java:319)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.access$700(ProjectManagerImpl.java:82)
	at com.intellij.openapi.project.impl.ProjectManagerImpl$12.compute(ProjectManagerImpl.java:629)
	at com.intellij.openapi.project.impl.ProjectManagerImpl$12.compute(ProjectManagerImpl.java:625)
	at com.intellij.openapi.progress.impl.CoreProgressManager$4.run(CoreProgressManager.java:208)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:563)
	at com.intellij.openapi.progress.impl.CoreProgressManager$8.run(CoreProgressManager.java:367)
	at com.intellij.openapi.progress.impl.CoreProgressManager$2.run(CoreProgressManager.java:152)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:452)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:402)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:54)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:137)
	at com.intellij.openapi.application.impl.ApplicationImpl$10$1.run(ApplicationImpl.java:617)
	at com.intellij.openapi.application.impl.ApplicationImpl$8.run(ApplicationImpl.java:400)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:695)
	at org.jetbrains.ide.PooledThreadExecutor$1$1.run(PooledThreadExecutor.java:56)
Caused by: com.intellij.diagnostic.PluginException: kotlin/PropertyMetadata [Plugin: Vue.js]
	... 29 more
Caused by: java.lang.NoClassDefFoundError: kotlin/PropertyMetadata
	at org.vuejs.VueJS.<clinit>(VueJS.kt)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:249)
	at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentsRegistry.a(ComponentManagerImpl.java:427)
	... 26 more
Caused by: java.lang.ClassNotFoundException: kotlin.PropertyMetadata PluginClassLoader[Vue.js, 1.0.1]
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:68)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:247)
	... 30 more
2016-01-03 23:46:43,412 [   7941]   INFO - ellij.project.impl.ProjectImpl - 16 project components initialized in 155 ms 
2016-01-03 23:46:43,412 [   7941]   INFO - le.impl.ModuleManagerComponent - 0 module(s) loaded in 0 ms 
2016-01-03 23:46:43,422 [   7951]   INFO - roject.impl.ProjectManagerImpl - com.intellij.diagnostic.PluginException: Could not initialize class org.vuejs.VueJS [Plugin: Vue.js] 
com.intellij.ide.plugins.PluginManager$StartupAbortedException: com.intellij.diagnostic.PluginException: Could not initialize class org.vuejs.VueJS [Plugin: Vue.js]
	at com.intellij.ide.plugins.PluginManager.handleComponentError(PluginManager.java:245)
	at com.intellij.openapi.components.impl.PlatformComponentManagerImpl.handleInitComponentError(PlatformComponentManagerImpl.java:39)
	at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentsRegistry.a(ComponentManagerImpl.java:441)
	at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentsRegistry.a(ComponentManagerImpl.java:417)
	at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentsRegistry.access$000(ComponentManagerImpl.java:403)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.a(ComponentManagerImpl.java:108)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.init(ComponentManagerImpl.java:87)
	at com.intellij.openapi.project.impl.ProjectImpl.init(ProjectImpl.java:284)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.a(ProjectManagerImpl.java:319)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.newProject(ProjectManagerImpl.java:271)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.newProject(ProjectManagerImpl.java:230)
	at com.intellij.platform.PlatformProjectOpenProcessor.doOpenProject(PlatformProjectOpenProcessor.java:193)
	at com.intellij.ide.RecentDirectoryProjectsManager.doOpenProject(RecentDirectoryProjectsManager.java:56)
	at com.intellij.ide.RecentProjectsManagerBase.doReopenLastProject(RecentProjectsManagerBase.java:357)
	at com.intellij.ide.RecentProjectsManagerBase$MyAppLifecycleListener.appStarting(RecentProjectsManagerBase.java:384)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at com.intellij.util.messages.impl.MessageBusConnectionImpl.deliverMessage(MessageBusConnectionImpl.java:116)
	at com.intellij.util.messages.impl.MessageBusImpl.doPumpMessages(MessageBusImpl.java:322)
	at com.intellij.util.messages.impl.MessageBusImpl.pumpMessages(MessageBusImpl.java:309)
	at com.intellij.util.messages.impl.MessageBusImpl.sendMessage(MessageBusImpl.java:288)
	at com.intellij.util.messages.impl.MessageBusImpl.access$200(MessageBusImpl.java:41)
	at com.intellij.util.messages.impl.MessageBusImpl$2.invoke(MessageBusImpl.java:177)
	at com.sun.proxy.$Proxy43.appStarting(Unknown Source)
	at com.intellij.idea.IdeaApplication$IdeStarter$2.run(IdeaApplication.java:328)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:332)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:209)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:715)
	at java.awt.EventQueue.access$400(EventQueue.java:82)
	at java.awt.EventQueue$2.run(EventQueue.java:676)
	at java.awt.EventQueue$2.run(EventQueue.java:674)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:86)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:685)
	at com.intellij.ide.IdeEventQueue.e(IdeEventQueue.java:734)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:569)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:382)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:296)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:211)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:196)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:188)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:122)
Caused by: com.intellij.diagnostic.PluginException: Could not initialize class org.vuejs.VueJS [Plugin: Vue.js]
	... 45 more
Caused by: java.lang.NoClassDefFoundError: Could not initialize class org.vuejs.VueJS
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:249)
	at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentsRegistry.a(ComponentManagerImpl.java:427)
	... 42 more
```

**Comment by morriswinkler** [comment]
+1

**Comment by ellisthedev** [comment]
This plugin is also broken in the latest versions of PhpStorm. Such a shame IntelliJ doesn't offer .vue support natively since VueJS is taking off. :(

**Comment by asolopovas** [comment]
+1

**Comment by kairos0ne** [comment]
OSX Phpstorm 2016 +1

**Comment by postalservice14** [comment]
Thanks.  I'm getting it too now.  I'll work on it.  If someone wants to
open a PR if they have a solution.  I welcome it.

**Comment by postalservice14** [comment]
I just released version 1.1.0.  Let me know if that fixes the problem.

**Comment by momo9** [comment]
Thanks. It works now.
